### ISSUE TITLE
JS OdinBook: Link is missing href

### ISSUE THREAD

**Issue by MaoShizhong** [TheOdinProject/curriculum#28321]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

In [JS OdinBook](https://www.theodinproject.com/lessons/nodejs-odin-book), there's a link to `socket.io` that's missing the href. That is, the text literally says `[socket.io]`. The lesson file should instead have `[socket.io](https://socket.io/)`, which will then render on the website as [socket.io](https://socket.io/).

## Acceptance criteria

Please comment below if you would like to be assigned to work on this issue. Please do not open a PR unless you have been assigned by a maintainer.

- [ ] Add socket.io href, so `[socket.io]` becomes `[socket.io](https://socket.io/)`

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/nodejs-odin-book

**Comment by AhmedTheGreatest** [comment]
Hello, I am willing to work on this issue. But should I work on this or let a person contribute for their first time?

**Comment by MaoShizhong** [comment]
@AhmedTheGreatest I'm happy to assign you, but if you'd prefer to leave it open for a less experienced contributor, then that's also fine. Let me.know what you'd like to do.

**Comment by JeyLson0** [comment]
Hi, first time contributor here, I want to work on this issue.

**Comment by AhmedTheGreatest** [comment]
@MaoShizhong Actually you can assign @JeyLson0 as he is a first time contributor.

**Comment by MaoShizhong** [comment]
All yours @JeyLson0. Let us know if you run into any problems.

**Comment by JeyLson0** [comment]
Thanks for the opportunity @AhmedTheGreatest.

@MaoShizhong already sent a PR, let me know if there are any issue.
